library/src/lib/menu/menu.component.scss

```diff
@@ -3,3 +3,7 @@
         outline: var(--fd-color-action-focus) dotted 1px;
     }
 }
+.fd-menu__list-separator > ul > li,
+.fd-menu__list-separator > fd-menu-group > ul > li{
+    border-bottom: 1px #EDEDED solid;
+}
```

Comment:
The General problem of this solve is that, we would need to follow all of the changes from fundamental styles. So let's use prepared class and delete this one.